_posts/2022-11-07-relicensing.md

```diff
@@ -0,0 +1,201 @@
+> represent MDAnalysis' full legal position on software licensing, it
+> purely aims to inform MDAnalysis developers and users on why
+> we believe the library should be relicensed.
+>
+> Further information on open-source software licensing can be found
+> from sources such as the [Open Source Initiative][OSI],
+> [tl;drLegal][tldr legal] and the [Software Sustainability Insitute][SSI licensing].
+>
+> Should you have any concerns about licensing, we always strongly
+> recommend getting legal advice before making any decisions on how
+> licensing changes may affect you.
+
+
+## Overview
+
+We want to **change the license of MDAnalysis** from the [GNU General
+Public License v2 (or any later versions)][GPLv2] (GPL v2+) to the less
+restrictive [Lesser GNU Lesser Public License v3 (or any later versions)][LGPL]
+(LGPL v3+) license. Both are [_open source licenses_][OSI] but it is
+our view that the [LGPL v3+][LGPL] will give developers more freedom
+in how they license any of their own codes that makes use of MDAnalysis.
+
+As detailed by the [Open Source Definition][OSD], licenses are core to
+the definition of open source. "Open source doesn't just mean access
+to the source code". The license defines how code can be used, copied,
+changed, and incorporated into other code.
+
+**License changes affect how folks interact with the MDAnalysis code
+base going forward. We need the agreement of our contributors and
+community members to change from GPL v2+ to LGPL v3+.**
+
+In this post we want to share the motivation, outline the relicensing
+process, and invite comments / questions from the community.
+
+
+## Rationale for license change
+
+
+### Why not GPL v2+ any more?
+
+Since its initial release in 2008, MDAnalysis has grown from a small
+Python package used by a handful of enthusiastic graduate students and
+postdocs to a mature library that is used by thousands of researchers
+in the molecular sciences. The MDAnalysis library was published under
+an open source license from the start so that anyone could freely use
+it, contribute to it, and build on it. We chose the GNU Public License
+version 2+ for this purpose. Thanks to the "copy-left" portion of the
+GPL v2+, which requires anyone using MDAnalysis in their own code to
+also adopt a compatible version of the GPL for their code, contributors
+could feel that any time and work that they donated and invested into
+MDAnalysis would not end up contributing to software under non
+open-source licensing.
+
+However, the GPL v2+ has also created barriers to adoption of MDAnalysis.
+It has, under our (and many other's) interpretation of the GPL v2+
+license, prevented developers from making their own code available under
+non-GPL licenses. Ultimately, it is the MDAnalysis core team's view that
+we do not want to dictate how our developers and users should license
+their code, but we also wish to ensure that work on the MDAnalysis
+library remains open and free.
```

Comment:
```suggestion
Under many interpretations, this license prevents
developers who use MDAnalysis from making their own code available under
non-GPL licenses. It is the MDAnalysis core team's view that
we *do not* want to dictate how our developers and users should license
their code, but we *do* wish to ensure that work on the MDAnalysis
library remains open and free.
```

Reply:
TODO - to add slightly updated version of this

Reply:
removing ensure to "wish that work on the..."